src/Expose/FilterCollection.php

```diff
@@ -88,6 +92,21 @@ public function setFilterData($data)
     }
 
     /**
+     * Alter the impact level of a specific filter id.
+     *
+     * @param integer $filterId
+     * @param integer $impact
+     */
+    public function setFilterImpact($filterId, $impact) {
+        $filter = $this->getFilterData($filterId);
+
+        if($filter === null) {
+            return;
```

Comment:
Actually, I think that was me, and not a merge issue.  I prefer to keep exit conditions clear and short.  While in this case, the "non-exit" is quite brief, if it was longer, I find that you have to understand more code to determine where it may have exited, rather than seeing the explicit condition.

It's also easier to see when you've missed paths in the code when doing code coverage (again, with more complex paths, not so much here)

Basically, though, it's just habit and convention for my day job, and I'll change it if it's wrong for Exposé.